app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,46 @@
+class PostsController < ApplicationController
+
+  
+ 
+  def show
```

Comment:
You're referring to the number of spaces separating "class" and "def show" right? 
I believe I have my default tab space to 2 spaces. Should I reduce that to one?